src/plugins/intel_gpu/tests/unit/fusions/softmax_fusion_test.cpp

```diff
@@ -71,6 +71,9 @@ class SoftmaxPrimitiveFusingTest : public ::BaseFusingTest<softmax_test_params>
 #define CASE_SOFTMAX_FP16_1 {1, 15, 4, 5}, data_types::f16, format::bfyx, 1, data_types::f16, format::bfyx
 #define CASE_SOFTMAX_FP16_2 {1, 15, 4, 5}, data_types::f16, format::bfyx, 3, data_types::f16, format::bfyx
 
+#define CASE_SOFTMAX_BF_FP32_1 {1, 15, 1, 1}, data_types::f32, format::bfyx, 1, data_types::f32, format::bfyx
```

Comment:
What is the intension of those CASE_SOFTMAX_BF? Please comment detail behavior of those CASE_SOFTMAX_BF.. test cases compared with existing ones.

Reply:
I also think "BF" is not a good naming here because it can be confused with bf16. 
I dont think we need a new name,... just CASE_SOFTMAX_FP32_XXX is enough.
BTW can these TCs cover the intended issue? (float input and i8 output)

Reply:
> I also think "BF" is not a good naming here because it can be confused with bf16. I dont think we need a new name,... just CASE_SOFTMAX_FP32_XXX is enough. BTW can these TCs cover the intended issue? (float input and i8 output)

The testcase names were modified as you commented. And yes, it was tested as expected and without it the test was failed. Thanks for your comment.

Reply:
> What is the intension of those CASE_SOFTMAX_BF? Please comment detail behavior of those CASE_SOFTMAX_BF.. test cases compared with existing ones.

The test case name was modified as Taylor commented. Thanks for your comment.